src/wp-admin/includes/post.php

```diff
@@ -663,6 +663,15 @@ function bulk_edit_posts( $post_data = null ) {
 		}
 	}
 
+	/**
+	 * Fires after the post data are processed for bulk edit.
```

Comment:
"data are" is officially correct since it's a plural noun, but I can imagine future tickets asking for it to be changed to "data is", an acceptable alternative when used for "data" (as in "dataset"). To save some ticket gardening/possible future patching, what about this?

> Fires after processing the post data for bulk edit.

Reply:
That's a very relevant comment, thanks!